pricing/views.py

```diff
@@ -118,7 +122,13 @@ def post(self, request):
         ResponseDict["discounts"] = usedDiscounts
         ResponseDict["discount_total"] = totalDiscountVal
         ResponseDict["price_adjustment"] = price_adjustment
-        ResponseDict["total"] = sub_total-totalDiscountVal-price_adjustment      
+        ResponseDict["total"] = sub_total-totalDiscountVal-price_adjustment
+
+        # parent balance adjustment
+        if body.get("apply_account_balance", False):
+            parent = Parent.objects.get(user_id=body["parent_id"])
+            ResponseDict["account_balance"] = float(parent.balance)
+            ResponseDict["total"] -= float(parent.balance)
```

Comment:
don't we also have to subtract the value applied from the parent balance?

Reply:
Do you mean line 131?

Reply:
This is price quote so we would not subtract from the parent's balance

Reply:
ah gotcha this is just the quote